src/ClientImpl.cpp

```diff
@@ -281,12 +284,18 @@ ClientImpl::ClientImpl(ClientConfig config) throw(voltdb::Exception, voltdb::Lib
     }
 #endif
     struct event_config *cfg = event_config_new();
+    if (!cfg)
+    {
+        throw voltdb::LibEventException();
+    }
     event_config_set_flag(cfg, EVENT_BASE_FLAG_NO_CACHE_TIME);//, EVENT_BASE_FLAG_NOLOCK);
     m_base = event_base_new_with_config(cfg);
     assert(m_base);
     if (!m_base) {
+	event_config_free(cfg);
         throw voltdb::LibEventException();
     }
+    event_config_free(cfg);
```

Comment:
The reference for cleanup of cfg was found here  http://www.wangafu.net/~nickm/libevent-book/Ref2_eventbase.html

To allocate an event_base with these functions, you call event_config_new() to allocate a new event_config. Then, you call other functions on the event_config to tell it about your needs. Finally, you call event_base_new_with_config() to get a new event_base. When you are done, you can free the event_config with event_config_free().